src/components/DataProductCard.tsx

```diff
@@ -121,30 +30,119 @@ const DataProductCard: React.FC<IProps> = ({
   description,
   data,
 }) => (
-  <Card hideArrow={true} to={url}>
-    <ImageWrapper background={background}>
-      <Image image={image} alt={alt ? alt : `${name} logo`} />
-    </ImageWrapper>
-    <Content>
-      <div>
-        <Title>{name}</Title>
-        {description && <Description>{description}</Description>}
-      </div>
+  <Link
+    href={url}
+    color="text"
+    background="searchBackground"
+    border="1px solid"
+    borderColor="lightBorder"
+    borderRadius="4px"
+    overflow="hidden"
+    boxShadow="table"
+    textDecoration="none"
+    display="flex"
+    flexDirection="column"
+    _hover={{
+      background: "tableBackgroundHover",
+      boxShadow: "tableBoxShadow",
+      transition: "transform 0.1s ease 0s",
+      transform: "scale(1.02)",
+    }}
+    isExternal
+    hideArrow
+  >
+    <Box
+      display="flex"
```

Comment:
Lets use `Flex` https://chakra-ui.com/docs/components/flex/usage
```suggestion
    <Flex
```

Reply:
I already did it.